lambda_functions/v1/Dockerfile-Function

```diff
@@ -0,0 +1,9 @@
+FROM public.ecr.aws/lambda/python:3.12
```

Comment:
Does this not have loads of trivy vulns? I seem to remember trying to use this image before but it had loads of junk come up on scan so used alpines for everything. If not then that's great as it's a lot easier